### PR TITLE
The Piano Has Been Drin- Broken! and made directional too

### PR DESCRIPTION
**Pull request by JMWTurner**
piano

When you push the piano, the keyboard will always be facing you.

Piano stools also appear above you when facing north now.

### CODE REVIEW

**Comment by SkowronX** [comment]
Somebody stop this madman.

**Comment by ihadtoregisterforthis** [comment]
The moog facing up looks weird

**Comment by ghost** [comment]
The moog facing up is making me think it's a mimic.

Moog mimic when?

File: code/game/objects/structures/stool_bed_chair_nest/stools.dm [hunk 1]
```diff
@@ -25,6 +25,11 @@
 	fire_fuel = 3
 	sheet_path = /obj/item/stack/sheet/wood
 
+/obj/item/weapon/stool/piano/New()
+	..()
+	spawn(3)
```
Comment (clusterfack) [inline code/game/objects/structures/stool_bed_chair_nest/stools.dm]: You should handle this via initialize() instead

File: code/game/objects/structures/stool_bed_chair_nest/chairs.dm [hunk 2]
```diff
@@ -9,10 +9,9 @@
 
 	lock_type = /datum/locking_category/chair
 
-/obj/structure/bed/chair/New()
+/obj/structure/bed/chair/initialize()
```
Comment (clusterfack) [inline code/game/objects/structures/stool_bed_chair_nest/chairs.dm]: Sorry there but new() should still call if(ticker) initialize()

Comment (JMWTurner) [reply code/game/objects/structures/stool_bed_chair_nest/chairs.dm:17]: I doubt it, I compiled and tested it in game on the chairs already facing north in the CMO's office appeared above me.

Comment (clusterfack) [inline code/game/objects/structures/stool_bed_chair_nest/chairs.dm]: >I doubt it

Can you, not do that?

Comment (JMWTurner) [reply code/game/objects/structures/stool_bed_chair_nest/chairs.dm:17]: do what? I asked DrCelt and he doesn't either understand what you mean here.

**Comment by ComicIronic** [comment]
The moog's north perspective is wrong.

File: code/game/objects/structures/stool_bed_chair_nest/chairs.dm (hunk 2 above)
Comment (clusterfack) [inline code/game/objects/structures/stool_bed_chair_nest/chairs.dm]: Not doubt me

**Comment by JMWTurner** [comment]
revision 2
- re-added chair/New()
- improved the Minimoog's north and south sprites.

moog

**Pull request merged**